crates/tx5-go-pion-sys/build.rs

```diff
@@ -319,3 +337,14 @@ fn gen_rust_const() {
         std::fs::write(&out, out_lines.join("\n")).unwrap();
     }
 }
+
+fn find_musl_gcc() -> std::path::PathBuf {
+    let output = Command::new("which")
+        .arg("musl-gcc")
+        .output()
+        .expect("musl-gcc not found");
```

Comment:
pretty linux specific, but i guess musl is by and large only used on linux anyways...

Reply:
Yes I'm currently only focused on `glibc` independence so binaries can be moved around between Linux distros that aren't on the very latest glibc like Nix is. I don't think macos has the same problem, we already fetch generic versions of libraries to work against for that... but I'll have to see when I get to that.